Bottom: a38876a7f86b91ae92a899377f6e1aca7e18d7b5
Top:    49ae73e5b2fff5d71d4ce9f3846b6d6724506e18
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-05-29 17:17:56 +0800

Refresh of vhost_net-introduce

---

diff --git a/drivers/vhost/net.c b/drivers/vhost/net.c
index b426686..f499aca 100644
--- a/drivers/vhost/net.c
+++ b/drivers/vhost/net.c
@@ -473,8 +473,8 @@ static size_t init_iov_iter(struct vhost_virtqueue *vq, struct iov_iter *iter,
 
 static bool vhost_exceeds_weight(int pkts, int total_len)
 {
-	return unlikely(total_len >= VHOST_NET_WEIGHT) ||
-	       unlikely(pkts >= VHOST_NET_PKT_WEIGHT);
+	return total_len >= VHOST_NET_WEIGHT ||
+	       pkts >= VHOST_NET_PKT_WEIGHT;
 }
 
 /* Expects to be always run from workqueue - which acts as
